homes-ms-storesmanagement/src/main/java/com/tenx/ms/storesmanagement/commons/rest/AbstractStoresManagementController.java

```diff
@@ -0,0 +1,40 @@
+package com.tenx.ms.storesmanagement.commons.rest;
+
+import com.tenx.ms.commons.rest.AbstractController;
+import org.springframework.dao.DataIntegrityViolationException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.converter.HttpMessageNotReadableException;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+/**
+ *
+ */
+public abstract class AbstractStoresManagementController extends AbstractController {
+
+    @ResponseStatus(value = HttpStatus.PRECONDITION_FAILED)
+    @ExceptionHandler(DataIntegrityViolationException.class)
+    protected void handleUpdateViolationException(DataIntegrityViolationException ex,
```

Comment:
I think you meant to use the exception you handled as the name of this method? Though this is not going to break anything.